gnome-initial-setup/gis-driver.c

```diff
@@ -204,6 +208,32 @@ gis_driver_get_user_language (GisDriver *driver)
 }
 
 void
+gis_driver_add_skipped_page (GisDriver   *driver,
+                             const gchar *page_id)
+{
+  GisDriverPrivate *priv = gis_driver_get_instance_private (driver);
+
+  if (gis_driver_is_skipped_page (driver, page_id))
+    return;
+
+  priv->skip_pages = g_list_prepend (priv->skip_pages, g_strdup (page_id));
+}
+
+gboolean
+gis_driver_is_skipped_page (GisDriver *driver, const gchar *page_id)
+{
+  GisDriverPrivate *priv = gis_driver_get_instance_private (driver);
+  GList *item;
+
+  for (item = priv->skip_pages; item; item = g_list_next (item)) {
+    if (g_strcmp0 ((char *)item->data, page_id) == 0)
+      return TRUE;
```

Comment:
It would provide direct access, sure, but this list is likely not going to be long anyway (typically one item), so I'm not sure it's worth. No strong opinions, though.